Sources/TokamakDemo/main.swift

```diff
@@ -17,11 +17,11 @@ import TokamakDOM
 
 let document = JSObjectRef.global.document.object!
 
-_ = document.head.object!.insertAdjacentHTML!("beforeend", #"""
-<link
-  rel="stylesheet"
-  href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
-"""#)
+// _ = document.head.object!.insertAdjacentHTML!("beforeend", #"""
+// <link
+//  rel="stylesheet"
+//  href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
+// """#)
```

Comment:
Let's remove it altogether if it's now unused.

```suggestion
// _ = document.head.object!.insertAdjacentHTML!("beforeend", #"""
// <link
//  rel="stylesheet"
//  href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
// """#)
```